### PR TITLE
Přechod na hosting ve Vercelu

### PR DESCRIPTION
**Pull request by zoul**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,19 +4,17 @@ Kód je uvolněn pod [licencí MIT](https://opensource.org/licenses/MIT). Na tex
 
 ## Technické řešení
 
-Web je statický, hostovaný na [Netlify](https://www.netlify.com) a generovaný pomocí [Jekylla](http://jekyllrb.com). Zde je uložené pouze HTML a související data, zejména tedy nikoliv fotky a větší data.
+Web je statický, hostovaný na [Vercelu](https://vercel.com) a generovaný pomocí [Jekylla](http://jekyllrb.com). Zde je uložené pouze HTML a související data, zejména tedy nikoliv fotky a větší data.
```
Comment (ckopecky) [inline README.md:7]: https://www.vercel.com?utm_source=[ohlasy]&utm_campaign=oss

**Correct link for Vercel sponsorship**

File: README.md [hunk 2]
```diff
@@ -27,3 +25,5 @@ Náhledový server vyžaduje autentizaci, aby ho nemohl používat kdokoliv mimo
 ### forum.ohlasy.info
 
 Virtuální server u [Digital Ocean](https://digitalocean.com), ručně instalovaná instance [Discourse](https://www.discourse.org).
+
+![Powered by Vercel](/assets/vercel.svg?raw=true)
```
Comment (ckopecky) [inline README.md:29]: Please make this a link that goes to https://www.vercel.com?utm_source=[ohlasy]&utm_campaign=oss

**for Vercel Sponsorship**

**Review by ckopecky — CHANGES_REQUESTED** [review]
Hi there! I left two corrections so that I can approve you for sponsorship for Vercel. :) Have a great day! I'll email through the support channel as well. Just let me know there when this PR has been merged

File: README.md (hunk 1 above)
Comment (zoul) [reply README.md]: Thank you, fixed!

File: README.md (hunk 2 above)
Comment (zoul) [reply README.md]: Thank you, fixed!

**Pull request merged**